### DISCUSSION TITLE
Release 1.4.10

### DISCUSSION THREAD

**Discussion by bsongis-frsky** [FrSkyRC/ETHOS-Feedback-Community#2821] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/FrSkyRC/ETHOS-Feedback-Community/releases/tag/1.4.10'>Release 1.4.10</a>.</em>

**Comment by moorslope** [comment]
where can i down load 1.4.11 as 1.4.12 will not run on my mac book pro

**Comment by bsongis-frsky** [comment]
In the releases section

**Reply by kitchenman** [reply]
Do i have to install 14.10 or can i jump to 14.14
